docs/modules/data-structures/pages/listening-for-map-entries.adoc

```diff
@@ -99,21 +110,37 @@ C++::
 ----
 --
 
-C Sharp:: 
+C Sharp::
 +
--- 
+--
 [source,cs]
 ----
 
 ----
 --
 
-Node.js:: 
-+ 
--- 
+Node.js::
++
+--
 [source,javascript]
 ----
-
+const client = await Client.newHazelcastClient();
+const map = await client.getMap('map');
+
+const listener = {
+    added: (entryEvent) => {
+        console.log('Entry added: ' + entryEvent.value);
```

Comment:
technically value is not entry. can you print both key and value with something like: (in backticks) `key: ${entryEvent.key}, value: ${entryEvent.value}`